### PR TITLE
Adding .json suffix to JSON files

### PR DESCRIPTION
**Pull request by 3uzbcqje**
Adding .json suffix to JSON files, to make these files display/function better in IDEs and other tools.

### CODE REVIEW

File: examples/confidential-coins/scripts/test_confidential_coins [hunk 1]
```diff
@@ -51,7 +51,7 @@ pushd _test_confidential_coins
     AGGREGATOR_KEYFILE="aggregator.key"
     SNARKJSVKFILE="../circuits/snarkjs_verification_key.json"
     APP_VKFILE="../circuits/upa_verification_key.json"
-    UPAINSTANCE="upa.instance"
+    UPAINSTANCE="upa.instance.json"
```
Comment (dtebbs) [inline examples/confidential-coins/scripts/test_confidential_coins:54]: Should we go the whole way and do `upa-instance.json`?  Or maybe `upa_instance.json`.
We probably have some inconsistencies re `-` vs `_`, but introducing `upa.instance.json` might look like yet another convention.

**Review by dtebbs — APPROVED** [review]
One comment about consistency, otherwise LGTM.
Also, are there no files that need to be renamed?

(Gitbook probably needs a PR ready for when this gets released)

**Comment by dtebbs** [comment]
> Also, are there no files that need to be renamed?
Actually, no I guess there wouldn't be.  We can't really check them in.

File: examples/confidential-coins/scripts/test_confidential_coins (hunk 1 above)
Comment (3uzbcqje) [reply examples/confidential-coins/scripts/test_confidential_coins:54]: Good question. We just need to pick one and do it. I don't think there's any significant differences... I think underscore is a pretty normal standard. I can update it to that.

**Pull request closed without merging**